msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -341,7 +342,10 @@ Use the following example to configure your app manifest's `webApplicationInfo`
 </details>
 
 > [!NOTE]
-> The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
+>
+> * The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
+>
+> * For one-on-one and group calls `organizer` is the initiator of the chat.
```

Comment:
Please note this is only true for one-on-one call. For group calls "organizer" is the call initiator